inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -505,11 +492,17 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
     if (!ngraphNode && opsets.count(params.version)) {
         auto opset = opsets.at(params.version);
 
-        if (!opset.contains_type(params.type)) {
+        // ROIPooling was missing in opset1 and was added in opset2
+        if (params.type == "ROIPooling" && params.version == "opset1") {
+            opset = opsets.at("opset2");
+        }
+
+        if (!opset.contains_type_insensitive(params.type)) {
```

Comment:
which op requires insensitive check?

Reply:
In this case it was PReLU

Reply:
Looks like there is a mistake in type name in functional test. I checked open model zoo example (https://download.01.org/opencv/2021/openvinotoolkit/2021.1/open_model_zoo/models_bin/2/person-reidentification-retail-0287/FP32/person-reidentification-retail-0287.xml) and type name for v10 is "PReLU".

Reply:
It is a correct change. Because nGraph op type is `Relu` but in the MO and in the spec we use `ReLU`.
We have some problems with case sensitive.